pyserini/eval/msmarco_doc_eval.py

```diff
@@ -0,0 +1,21 @@
+import argparse
+import subprocess
+from pyserini.util import download_evaluation_script
+
+parser = argparse.ArgumentParser(description='Evaluate msmarco doc runs.')
+parser.add_argument('--qrel', type=str, required=True, help="Path to qrel file")
+parser.add_argument('--run', type=str, required=True, help="Path to run file")
+args = parser.parse_args()
+
+script_path = download_evaluation_script('msmarco_doc_eval')
+cmd = ['python3', script_path, '--judgments', args.qrel, '--run', args.run]
```

Comment:
I don't think we should try to parse and rename the args. Just pass unchanged to underlying eval script.